tests/integration/ceph_csi_test.go

```diff
@@ -0,0 +1,208 @@
+	keyResult, err := k8sh.Exec(namespace, "rook-ceph-tools", "bash", commandArgs)
+	logger.Infof("Ceph get-key: %s", keyResult)
+	require.Nil(s.T(), err)
+	commandArgs = []string{"-c", "ceph mon_status"}
+	monResult, err := k8sh.Exec(namespace, "rook-ceph-tools", "bash", commandArgs)
+	logger.Infof("Ceph mon_status: %s", monResult)
+	require.Nil(s.T(), err)
+
+	var mon cephMonStatus
+	err = json.Unmarshal([]byte(monResult), &mon)
+	require.Nil(s.T(), err)
+	require.True(s.T(), len(mon.Monmap.Mons) > 0, "no mon found")
+	monStr := strings.Split(mon.Monmap.Mons[0].PublicAddr, "/")[0]
+	require.True(s.T(), len(monStr) > 0, "invalid mon addr")
+
+	_, err = k8sh.Clientset.Core().Secrets(namespace).Create(&v1.Secret{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      csiSecretName,
+			Namespace: namespace,
+		},
+		Data: map[string][]byte{
+			"admin":    []byte(keyResult),
+			"adminID":  []byte("admin"),
+			"adminKey": []byte(keyResult),
+			"monitors": []byte(monStr),
+		},
+	})
+	require.Nil(s.T(), err)
+	logger.Info("Created Ceph CSI Secret")
+}
+
+func createCephPools(helper *clients.TestClient, s suite.Suite, namespace string) {
+	out, err := helper.PoolClient.Create(csiPoolRBD, namespace, 1)
+	logger.Infof("rbd pool create: %+v", out)
+	require.Nil(s.T(), err)
+
+	out, err = helper.PoolClient.Create(csiPoolCephFS, namespace, 1)
+	logger.Infof("cephFS pool create: %+v", out)
+	require.Nil(s.T(), err)
+}
+
+func createCSIStorageClass(k8sh *utils.K8sHelper, s suite.Suite, namespace string) {
+	rbdSC := `
+apiVersion: storage.k8s.io/v1
+kind: StorageClass
+metadata:
+   name: ` + csiSCRBD + `
+provisioner: csi-rbdplugin
+parameters:
+    monValueFromSecret: "monitors"
+    pool: ` + csiPoolRBD + `
+    csi.storage.k8s.io/provisioner-secret-name: ` + csiSecretName + `
+    csi.storage.k8s.io/provisioner-secret-namespace: ` + namespace + `
+    csi.storage.k8s.io/node-publish-secret-name: ` + csiSecretName + `
+    csi.storage.k8s.io/node-publish-secret-namespace: ` + namespace + `
+    adminid: admin
+    userid: admin
+`
+	cephFSSC := `
+apiVersion: storage.k8s.io/v1
+kind: StorageClass
+metadata:
+   name: ` + csiSCCephFS + `
+provisioner: csi-cephFSplugin
+parameters:
+    monValueFromSecret: "monitors"
+    pool: ` + csiPoolCephFS + `
+    csi.storage.k8s.io/provisioner-secret-name: ` + csiSecretName + `
+    csi.storage.k8s.io/provisioner-secret-namespace: ` + namespace + `
+    csi.storage.k8s.io/node-stage-secret-name: ` + csiSecretName + `
+    csi.storage.k8s.io/node-stage-secret-namespace: ` + namespace + `
+    adminid: admin
+    userid: admin
+`
+	out, err := k8sh.ResourceOperation("apply", rbdSC)
+	logger.Infof("create rbd storage class: %v", out)
+	require.Nil(s.T(), err)
+
+	out, err = k8sh.ResourceOperation("apply", cephFSSC)
+	logger.Infof("create cephfs storage class: %v", out)
+	require.Nil(s.T(), err)
+}
+
+func createCSIRBDTestPod(k8sh *utils.K8sHelper, s suite.Suite, namespace string) {
```

Comment:
Could we get a test pod created for cephfs plugin as well? In a future PR is fine if needed.

Reply:
sure, will do.